reviews/tests.py

```diff
@@ -76,6 +79,16 @@ def setup_proposal(self):
         )
         self.proposal.generate_pdf()
 
+        self.pre_assessment = Proposal.objects.get(pk=2)
+        self.pre_assessment.applicants.add(self.user)
+        self.pre_assessment.wmo = Wmo.objects.create(
+            proposal=self.pre_assessment,
+            metc=YesNoDoubt.NO,
+        )
+        self.pre_assessment.supervisor = self.supervisor
+        self.pre_assessment.relation = Relation.objects.get(pk=4)
```

Comment:
Try to avoid just setting an attribute by PK like this, it tells the future programmer nothing. A comment indicating which relation we're getting can help, but even better would be:

```python3
self.phd_student_relation = Relation.objects.get(pk=4)
self.pre_assessment.relation = self.phd_student_relation
```

This is informative without needing a comment, and allows future tests to reuse this variable.